### ISSUE TITLE
When I drag a file, the visual studio code stops.

### ISSUE THREAD

**Issue by mihilt** [microsoft/vscode#105851]
- VSCode Version: I tried this in visual studio stable version  and insider version.

- OS Version: Microsoft Windows 10 Education

Steps to Reproduce:

I searched a lot, but I couldn't find an issue in the same situation as me.

About three months ago, it took a long time to change the location of files on the tab through drag.
There was also the same problem with repositioning files in 'explorer'.
Since then, it has not worked perfectly. When I drag a file, the visual studio code stops.

I thought this was due to the clash of many extentions. So I deleted all the extentions.
But the problem still existed.

I reset all the setup files of Visual Studio, deleted and re-installed.
But the problem still existed.

I installed insiders version. And I tried, but it still didn't work.

This works well in my laptop. But I don't know why my desktop computer is like this.

![GIF 2020-09-02 오전 1-19-04](https://user-images.githubusercontent.com/50368758/91880927-b650a780-ecbb-11ea-8154-34b7ea103fa2.gif)

It works as shown above.
Please help me.

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
I can not reproduce this.
Strange that you also see this with VS Code insiders.
Let's leave it open for now to see if other users hit this.

**Comment by mihilt** [comment]
I had something to do yesterday, so I wanted to use the visual studio code quickly.
So I made various attempts, and finally solved the problem.
I don't know what the exact solution is.
But the ways I did are as follows.

I've deleted all the strange programs, all the unknown services.

This is just my guess, but it seems to be due to a clash between security-related services and visual studio codes.

I searched a lot yesterday. But I couldn't find anyone who met the same problem as me. This is a special situation that happened only on my computer. So I don't think anyone else needs to waste time trying to solve the problem anymore.

Thanks for caring.
